index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>CV</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header>
+    <h1>Vladislav Lisetskii</h1>
+    <hr>
```

Comment:
it is not an outdated tag, to be honest, but it's usage is kind of discouraged :)
please use border-top or border-bottom css rules instead

Reply:
ok, thanks